dependencies.yaml

```diff
@@ -141,7 +188,9 @@ dependencies:
         packages:
           - numba>=0.57
           - numpy>=1.21
-          - *cuda_python
+    specific:
+      - output_types: [conda, requirements, pyproject]
+        <<: *cuda_python_matrix
```

Comment:
In case anyone is wondering, the `<<` is [a YAML merge key]( https://yaml.org/type/merge.html )

Reply:
The purpose is to repeat the `cuda-python` matrix from the build requirements to the run requirements without writing it out twice.

Reply:
This didn't work, see https://github.com/rapidsai/rmm/pull/1223#discussion_r1235996234. I'll resolve this thread since the info is no longer relevant.